rootex/core/renderer/shaders/custom_post_processing.hlsl

```diff
@@ -0,0 +1,22 @@
+Texture2D<float4> Texture : register(t0);
```

Comment:
rename file to something more meaningful

Reply:
fixed